plugins/pagerduty/package.json

```diff
@@ -68,5 +68,6 @@
   },
   "files": [
     "dist"
-  ]
+  ],
```

Comment:
Remember to add `config.d.ts` to the `files` section too!

Reply:
oof; good call out! made this same mistake when I added a custom schema to our own instance :) thanks